lib/rest-connector.js

```diff
@@ -133,10 +135,10 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
 exports.RestConnector = RestConnector;
 
 /**
- * The RestConnector constructor
+ * @class
+ * @classdesc The RestConnector constructor
```

Comment:
This is not a description of the RestConnector; it should be a short summary of what the RestConnector does (as an object).
Additionally, you can mark this JSDoc with the `@constructor` tag if you wish.